### PR TITLE
Swift 3 compatibility

### PR DESCRIPTION
**Pull request by RamonGilabert**
**Do not merge.**

### CODE REVIEW

**Comment by zenangst** [comment]
Shouldn't it just be a branch without a PR until further notice?

File: Example/BarcodeScannerExample/BarcodeScannerExample/ViewController.swift [hunk 1]
```diff
@@ -4,20 +4,20 @@ import BarcodeScanner
...
-    button.backgroundColor = UIColor.blackColor()
-    button.titleLabel?.font = UIFont.systemFontOfSize(28)
-    button.setTitleColor(UIColor.whiteColor(), forState: .Normal)
-    button.setTitle("Scan", forState: .Normal)
-    button.addTarget(self, action: #selector(buttonDidPress), forControlEvents: .TouchUpInside)
+    let button = UIButton(type: .system)
+    button.backgroundColor = UIColor.black
+    button.titleLabel?.font = UIFont.systemFont(ofSize: 28)
+    button.setTitleColor(UIColor.white, for: UIControlState())
+    button.setTitle("Scan", for: UIControlState())
```
Comment (vadymmarkov) [inline Example/BarcodeScannerExample/BarcodeScannerExample/ViewController.swift]: `.Normal` -> `UIControlState`?

**Comment by onmyway133** [comment]
@RamonGilabert I think `[WIP]` in the title should be nice

**Comment by RamonGilabert** [comment]
It's a PR to check if the branch is correct and what has changed and stuff. I can close it. Just for you to know what changes with Swift 3.

**Comment by oniikal3** [comment]
Anyone know? How can I update to swift 3 ?

**Comment by zenangst** [comment]
@oniikal3 you can modify your pod file to point to this Swift 3 branch.

```
pod 'BarcodeScanner', git: 'https://github.com/hyperoslo/BarcodeScanner', branch: 'swift-3'
```

**Comment by RamonGilabert** [comment]
Closing the PR since we've seen it already! :)

**Pull request closed without merging**